performance-test/README.md

```diff
@@ -0,0 +1,148 @@
+### Contains
+* This folder contains scenario-based performance test scripts for Resident Services.
+   
+*List of  Scenarios: 
+	* S01 UIN Services Login
+	* S02 View My History 
+	* S03 Manage My VID
+	* S04 Secure My ID 
+	* S05 Track My Requests
+	* S06 Get Personalized Card
+	* S07 Share My Data
+	* S08 Update My Data
+	* S09 Menu Bar
+	* S10 Profile ICon and Logout
+	* S11 Get My UIN
+	* S12 Get Information
+	* S13 Audit
+ 	* S14 Verify Phone/Email ID
+
+ 
+* Open source tools used,
+    1. [Apache JMeter](https://jmeter.apache.org/)
+
+### How to run performance scripts using Apache JMeter tool
+* Download Apache JMeter from https://jmeter.apache.org/download_jmeter.cgi
+* Download scripts for the required module.
+* Place the support files in the bin folder of the jmeter, the paths in the scripts are defined to fetch the testdata from the bin folder.
+* Start JMeter by running the jmeter.bat file for Windows or jmeter file for Unix. 
+* Validate the scripts for one user.
+* Execute a dry run for 10 min.
+* Execute performance run with various loads in order to achieve targeted NFR's.
+
+### Setup points before execution
+
+* We need some jmeter plugin files that needs to be insatlled before opening of this script, PFA dependency links for your reference : 
```

Comment:
insatlled - spell check

Reply:
Also little more details on how to install will help people

Reply:
1. Spell checks corrected
2. Added detailed information on how to install the plugins